packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1923,9 +2187,9 @@ class _MatchesSemanticsData extends Matcher {
   final AttributedString? attributedDecreasedValue;
   final String? tooltip;
   final SemanticsHintOverrides? hintOverrides;
-  final List<SemanticsAction>? actions;
+  final Map<SemanticsAction, bool> actions;
```

Comment:
Maybe leave a comment here indicating that this map and the one below indicate a tri-state situation: if an action maps to true it must be present, if an action maps to false, it must be absent. If an action is not in the map, there's no opinion on whether it needs to be present or not.

(Maps that map to boolean always urge me to replace them with a Set, but that's obviously not possible here because of the tri-state situation)

Reply:
Good call. Added a comment, and I also moved them down below the more trivial members for readability.